### PR TITLE
Avoiding breaking chain.

### PR DESCRIPTION
**Pull request by ecoslado**
@carlosescri. I just want you to have a look. Are these the kind of changes you need?

### CODE REVIEW

**Comment by carlosescri** [comment]
Just for recording this somewhere... This PR is going to increment version to 5.1.0 due to the change in dfdom methods and the way they work.

**Comment by ecoslado** [comment]
@carlosescri I added tests to this PR, so I can continue testing meanwhile you check it.

File: lib/util/dfdom.js [hunk 1]
```diff
@@ -61,13 +77,14 @@ author: @ecoslado
       });
       parent = null;
       while (el) {
+        console.log(el);
```
Comment (carlosescri) [inline lib/util/dfdom.js]: Notice there's a console.log here... not compiling?

**Comment by ecoslado** [comment]
@carlosescri maybe I uploaded without compiling. right now, no console.log

**Comment by ecoslado** [comment]
@carlosescri 

* dfdom stuff you requested for is done.

* dfdom tests.

please have a look. You'll see there's a grunt task to execute karma tests:
`$ grunt test_karma`

**Pull request merged**